content/dk/mada/style/mada-style-gradle/mada-style-gradle-1.0.0.buildspec

```diff
@@ -0,0 +1,38 @@
+# see instructions https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/doc/BUILDSPEC.md
+
+# Central Repository coordinates for the Reference release (for multi-module, pick an arbitrary module)
+groupId=dk.mada.style
+artifactId=mada-style-gradle
+version=1.0.0
+# referenceRepo = https://my.company.com/repo if reference is not in Maven Central but company-private
+
+display=${groupId}:${artifactId}
+
+# Source code
+gitRepo=https://github.com/jskov/${artifactId}.git
+gitTag=${version}
+#sourceDistribution=https://archive.apache.org/dist/maven/scm/${artifactId}-${version}-source-release.zip
+#sourcePath=${artifactId}-${version}
+#sourceRmFiles=DEPENDENCIES LICENSE NOTICE
+
+# Rebuild environment prerequisites
+tool=gradle
+# or tool=mvn-3.8.5 if default 3.6.3 version does not match your prerequisites (available version may be limited by images available on Dockerhub)
+# or tool=gradle or tool=sbt
+jdk=21
+newline=lf
+# crlf for Windows, lf for Unix
+umask=022
+#timezone="Etc/GMT"
+#locale="en_US"
+
+# Rebuild command
+command="bash ./gradlew --no-daemon javadoc publish publishToMavenLocal -Pversion=${version}"
+#command="bash ./gradlew --no-daemon clean assemble generateBuildInfo publishToMavenLocal -Pversion=${version}"
+
+# Location of the buildinfo file generated during rebuild (by attifact:compare for Maven) to record output fingerprints
+buildinfo=${artifactId}-${version}.buildinfo
+
+# if the release is finally not reproducible, link to an issue tracker entry if one was created or diffoscope result
+diffoscope=${artifactId}-${version}.diffoscope
```

Comment:
this part has to be commented out, as there is no issue in your rebuild

Reply:
I assume just the diffoscope line (I left issue= which is also present in reproducible-gradle)